springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/types/channel/operation/message/Message.java

```diff
@@ -10,7 +10,8 @@
  */
 @Data
 @Builder
-@EqualsAndHashCode(of = {"name"})
+// TODO Why ignore other fields?
+//@EqualsAndHashCode(of = {"name"})
```

Comment:
I'm not sure what the motivation of changing equals and hashcode was, but I'd rather do a custom equals (based on name) where it is needed instead of changing it for all other code because this causes other issues (ie test don't break)